Readme.md

```diff
@@ -4,81 +4,40 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# How to connect different data models to several databases within a single application
-
-> **Note**  
-> The description of this example is currently under construction and may not match the code in the example. We are currently working to provide you with up-to-date content.
-
-## Scenario
+# How to connect different data models to several databases within a single application with Entity Framework Core
 
 This example demonstrates how to create custom XAF modules with custom business objects and logic that would work with separate databases. These modules do not depend on each other and thus can be reused in other applications as a whole.  
```

Comment:
1. `This example demonstrates how to create custom XAF modules with custom business objects and logic that would work with separate databases.`

Can you change "that would work with separate databases" to "that work with separate databases"?

2. `These modules do not depend on each other and thus can be reused in other applications as a whole.`

By "as a whole" do you mean "independently"?

Reply:
I applied both suggestions, thank you!

Reply:
Thank you!